adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -1156,18 +1157,20 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
    * @return Returns a FeatureRDD.
    */
   def loadGff3(filePath: String,
+               cache: Boolean = true,
```

Comment:
Ah, so sorry if I'm being thick, we should pass the StorageLevel around the feature APIs instead, and then replace the call to `rdd.cache` in `FeatureRDD` with `rdd.persist(storageLevel)`?